Dockerfile

```diff
@@ -6,14 +6,16 @@ RUN apt-get update && \
     apt-get install -y git && \
     apt-get clean
 
-#Install main program
-COPY /src /app/src
+# Installing dependencies 
 COPY requirements.txt /app
 
 RUN python3.10 -m pip install -r requirements.txt
 ENV PYTHONUNBUFFERED=1
 
-LABEL org.opencontainers.image.source https://github.com/SolaceLabs/solace-ai-connector
+#Install main program
+COPY /src /app/src
+
+LABEL org.opencontainers.image.source=https://github.com/SolaceLabs/solace-ai-connector
```

Comment:
This is fine, but I am curious why the change?

Reply:
So the spaced key/value is the legacy format, in the newer docker files all the ARG, LABEL, and ENV should be `key=value`, More: https://docs.docker.com/reference/build-checks/legacy-key-value-format/

Reply:
If asking about why moving the COPY source code after the dependency installation, it's to best use the docker caching system, (Useful when using Docker for local development, no effect on CI/CD pipeline). Docker caches each stage (each line from the dockerfile) until the file being updated has changed, copying the source code after dep install allows not having to wait for dep installatIon after only changing the source code.
More: https://docs.docker.com/build/cache/#order-your-layers